frontend/templates/component/displayMessages.ts

```diff
@@ -0,0 +1,8 @@
+const messages = {
+    messageID: {
+        defaultMessage: 'A translatable string.',
+        description: 'A description of the translatable string.',
```

Comment:
Is this used or part of the template code?

Reply:
This is used to generate a sample file for a new component. I've added stuff to it to make it easier to understand.